Gemfile

```diff
@@ -3,4 +3,4 @@ source "https://rubygems.org"
 gemspec
 
 # Optional: use the development version of openxml-package
-# gem "openxml-package", github: "openxml/openxml-package", branch: "master"
+gem "openxml-package", github: "openxml/openxml-package", branch: "finish-requirements"
```

Comment:
Now that that's merged, let's change this back.

Reply:
I'd actually love to up the dependency number in the `gemspec` file and re-comment this line out once we cut a new version of the `openxml-package` gem.